src/features/canvassAssignments/components/PlaceDialog/pages/PlaceVisitPage.tsx

```diff
@@ -76,6 +76,7 @@ const PlaceVisitPage: FC<Props> = ({
                 }))
               );
               setSubmitting(false);
+              onClose();
```

Comment:
I think that somewhere there is also a `back()` (maybe inside of `onLogVisit()`?) that should probably be removed. Does that make sense?